k8s/predict/train-job/values-prod.yaml

```diff
@@ -3,7 +3,7 @@ app:
   namespace: production
   configmap: env-train-job-production
 
-schedule: 0 5 * * 1
+schedule: 0 1 1 * *
```

Comment:
[Schedule train job to run on the 1st of every month at 01:00 UTC](https://github.com/airqo-platform/AirQo-api/pull/1333/commits/ba10ed140f54b9fc1671a4f422a7798d5fc86499)